src/publishers/snapcraft.js

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs-extra';
+import path from 'path';
+import Snapcraft from 'electron-installer-snap/snapcraft';
+
+import asyncOra from '../util/ora-handler';
+
+/**
+ * `forgeConfig.snapStore`:
+ * * `release`: comma-separated list of channels to release to
+ */
+export default async ({ dir, artifacts, forgeConfig }) => {
+  const snapArtifacts = artifacts.filter(artifact => artifact.endsWith('.snap'));
+
+  if (snapArtifacts.length === 0) {
+    throw 'No snap files to upload. Please ensure that "snap" is listed in the "make_targets" in Forge config.';
+  }
+
+  const snapcraftCfgPath = path.join(dir, '.snapcraft', 'snapcraft.cfg');
+
+  if (!await fs.pathExists(snapcraftCfgPath)) {
+    throw `Snapcraft config not found, expected to be at "${snapcraftCfgPath}".`;
```

Comment:
I know the file is a .cfg, but what do you think of calling this "Snapcraft credentials" instead of "Snapcraft config"? And it would be nice to mention in the error that you can generate this file with snapcraft export-login.

Reply:
Sounds good.

Reply:
How's this: https://github.com/electron-userland/electron-forge/pull/427/commits/ad7120e5c5b01464b449de0acbd3227c3bf57aca

Reply:
Also, typo fixed.